surfsense_backend/app/connectors/jira_connector.py

```
@@ -0,0 +1,487 @@
+        """
+        try:
+            # Build JQL query for date range
+            # Query issues that were either created OR updated within the date range
+            date_filter = (
+                f"(createdDate >= '{start_date}' AND createdDate <= '{end_date}')"
+            )
+            # TODO : This JQL needs some improvement to work as expected
+
+            jql = f"{date_filter}"
+            if project_key:
+                jql = (
```

Ruff (0.12.2)

Local variable `jql` is assigned to but never used

Remove assignment to unused variable `jql`

(F841)